rust/ql/test/library-tests/dataflow/local/CONSISTENCY/DataFlowConsistency.expected

```diff
@@ -1,2 +0,0 @@
-identityLocalStep
-| main.rs:442:9:442:20 | phi(default_name) | Node steps to itself |
```

Comment:
I can see corresponding results - a big decrease in data flow inconsistencies - in the DCA run for Rust. ✨ 

The Swift DCA run shows an analysis slowdown, which is likely noise, but big enough I wouldn't mind a re-run to confirm nothing is wrong there.  Other than than the Rust and Swift DCA runs LGTM.

Reply:
The swift analysis slowdowns appear to be mostly on the order of the corresponding stddev values, so I expect it to be noise. But I've started a re-run, so we get another round of numbers.

Reply:
Hmm, looks like the Swift slowdown is at least somewhat consistent.

Reply:
I cannot reproduce any perf issues locally.

Reply:
In my experience, the Swift DCA analysis timings are completely useless (wdyt @redsun82 ?)

Reply:
unfortunately it's hard to get consistent timings from macOS runners, so I think those can be ignored

Reply:
The slowdowns being in different projects is good evidence this is noise to me.  :+1: